examples/plotting/airports_map.py

```diff
@@ -34,20 +36,8 @@ def plot(tile_source: TileSource):
     return p
 
 
-# create a tile source
-mq_tile_source = WMTSTileSource(
-    url="https://tiles.stadiamaps.com/tiles/stamen_terrain/{z}/{x}/{y}{r}.png",
-    extra_url_vars=dict(r=""),  # or "@2x" for 2x scaled (Retina) images
-    attribution="""
-&copy; <a href="https://stadiamaps.com/" target="_blank">Stadia Maps</a>
-&copy; <a href="https://stamen.com/" target="_blank">Stamen Design</a>
-&copy; <a href="https://openmaptiles.org/" target="_blank">OpenMapTiles</a>
-&copy; <a href="https://www.openstreetmap.org/about/" target="_blank">OpenStreetMap contributors</a>
-""",
-)
-
 carto = plot("CartoDB Positron")
-mq = plot(mq_tile_source)
+mq = plot(providers["OpenTopoMap"])
```

Comment:
Sorry, I meant something like:

https://github.com/bokeh/bokeh/blob/f16c29694584cee40253c41c587f367343fc5e14/examples/topics/geo/tile_xyzservices.py#L20

Reply:
Maybe this:
```python
In [1]: import xyzservices as xyz

In [2]: xyz.providers.OpenTopoMap
Out[2]:
{'url': 'https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png',
 'max_zoom': 17,
 'html_attribution': 'Map data: &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)',
 'attribution': 'Map data: (C) OpenStreetMap contributors, SRTM | Map style: (C) OpenTopoMap (CC-BY-SA)',
 'name': 'OpenTopoMap'}
```

Reply:
I see. I will change it to `xyz.OpenTopoMap`.